pkg/sysfs/system.go

```diff
@@ -70,7 +70,7 @@ type System struct {
 // Package is a physical package (a collection of CPUs).
 type Package struct {
 	id    ID    // package id
-	cpus  IDSet // CPUs in this package
+	Cpus  IDSet // CPUs in this package
```

Comment:
Why is this made a publid member? Either all of them should be public or none?

Reply:
I can't recall why already. I suspect the reason not to make all public is to reduce the patch size.

Reply:
But, do you need to make this public? I don't think you do

Reply:
This needs to be public to set up unit tests. Otherwise there's no way to create a policy with a mocked system in https://github.com/intel/cri-resource-manager/pull/90/files#diff-3e20512209e1f47a7250162092dbd6b5R47.

Reply:
I have another attempt at refactoring this file coming up soon, stay tuned. Maybe this work can be rebased on top of it.